app/scripts/components/common/cookie-consent/cookieConsent.spec.js

```diff
@@ -36,11 +54,13 @@ describe('Cookie consent form should render with correct content.', () => {
 
   it('Check correct cookie initialization', () => {
     const resultCookie = document.cookie;
-
     expect(resultCookie).toBe(
       `${COOKIE_CONSENT_KEY}={"responded":false,"answer":false}`
     );
   });
+  it('Check for session initialization', () => {
```

Comment:
I feel like this session value deserves more testing (it would have helped me understand what this session value is trying to do!) - Can we elaborate it?